lib/athena/index.js

```diff
@@ -56,12 +56,31 @@ class AthenaQueryClient {
   }
 
   /**
-   * Retrieves results from Athena once they are ready
+   * Recursively retrieves results from Athena once they are ready
    *
-   * @param {} queryInfo
+   * @param {object} queryInfo
+   * @param {array}  accumulatedResults
+   * @return {Promise.<object>}
    */
-  getResults(queryInfo) {
-    return this.athena.getQueryResults(queryInfo).promise();
+  getResults(queryInfo, accumulatedResults = []) {
```

Comment:
maybe we should not delete the original one and have a separate function. not sure if anyone else is using this

Reply:
That is a good point